OpenSim/Common/Storage.h

```diff
@@ -191,8 +195,8 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Storage, StorageInterface);
     int getDataColumn(const std::string& columnName,double *&rData) const;
     void getDataColumn(const std::string& columnName, Array<double>& data, double startTime=0.0) override;
 
-    /** Get a TimeSeriesTable out of the Storage.                             */
-    TimeSeriesTable getAsTimeSeriesTable() const;
+    /** Convert to a TimeSeriesTable (new in 4.0).                            */
```

Comment:
I don't think methods have been marked "new in 4.0" anywhere else. If it's necessary to draw the user's attention to this method, perhaps add a note in CHANGELOG?

Reply:
I was anticipating the user wondering "why would I want a TST over a Storage?" The "4.0" bit was to say "you might want a TST if you have to work with other new features in 4.0." Also, it was an attempt to subtly say that TST is the way of the future.

How about the following alternative: "Convert to a TimeSeriesTable. This may be useful if you need to use parts of the API that require a TimeSeriesTable instead of a Storage. We encourage you to use TimeSeriesTables instead of Storages if possible."

Reply:
Ah, ok. I suggest adding the full explanation, as "new in 4.0" didn't convey that information to me. If you use the text in your second paragraph, I suggest also explaining why "We encourage you to use TimeSeriesTables instead of Storages if possible" so we're not just telling the user to blindly do what we tell them to. I think the statement in your first paragraph ("you might want a TST if you have to work with other new features in 4.0") would be helpful.